src/common/utils/api/api.ts

```diff
@@ -86,8 +100,22 @@ async function apiGet<T>({ path, parameters = {} }: GetParameters): Promise<T> {
   });
 }
 
+async function apiPost<T>({ path, data = {} }: PostParameters): Promise<T> {
+  return request<T>({
+    url: `${apiBaseUrl}/v1${path}/`,
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    method: 'post',
+    data,
+    validateStatus(status) {
+      return status < 300;
+    },
+  });
+}
+
 export interface Resource {
```

Comment:
Could this be moved to the under lib/types?

Reply:
- [ ] Move interface Resource under lib/types